twiggy/tests/all/expectations/dominators_wee_alloc_with_depth_and_row

```diff
@@ -1,6 +1,6 @@
  Retained Bytes │ Retained % │ Dominator Tree
-────────────────┼────────────┼──────────────────────────────────────────────────────
+────────────────┼────────────┼────────────────────────────────────────────────────
             777 ┊     27.58% ┊ "function names" subsection
-            387 ┊     13.74% ┊ func[2]
-            378 ┊     13.42% ┊   ⤷ wee_alloc::alloc_with_refill::hb32c1bbce9ebda8e
+            387 ┊     13.74% ┊ wee_alloc::alloc_with_refill::hb32c1bbce9ebda8e
+            226 ┊      8.02% ┊   ⤷ wee_alloc::alloc_first_fit::h9a72de3af77ef93f
```

Comment:
Here's the test case that I think demonstrates this change most succinctly. Now, the topmost node contains the name of the function/code. The immediate child is another function that this dominates, as opposed to the code node that was displayed before.